src/app/state/playback_state.rs

```diff
@@ -407,13 +419,21 @@ impl From<PlaybackAction> for AppAction {
 pub enum PlaybackEvent {
     PlaybackPaused,
     PlaybackResumed,
+    Repeat(RepeatMode),
```

Comment:
just a minor thing, but for consistency I'd prefere RepeatModeChanged or something similar

Reply:
No problem will change